src/XsltProcessor.php

```diff
@@ -204,7 +162,8 @@ private function getTranspiledStyleSheet(Transpiler $transpiler, \DOMDocument $s
         $streamContext = \stream_context_create($this->createStreamOptions($transpiler));
         \libxml_set_streams_context($streamContext);
 
-        return $this->createTranspiledDocument($styleSheet);
+        $transpiledDocument = $this->createTranspiledDocument($styleSheet);
+        return $transpiledDocument;
```

Comment:
Could you merge this line into one

Reply:
Would you like me to try other approach? Some ideas:

- Instead of `Closure`, call a private method (i. e. `transformToXmlUnsafe`) and it would show the error location, `transformToXml` would be more like a envelop before calling the private method.

- Break `catchLibXmlErrorThrowTransformException` into `startCatchingLibXmlError` and `stopCatchingLibXmlError`.

- Revert de commit that creates `catchLibXmlErrorThrowTransformException` and just replace `\set_error_handler` with `\restore_error_handler` as needed.